### PR TITLE
Fix duplicated notification bitbucket

### PR DESCRIPTION
**Pull request by DannyFeliz**

### CODE REVIEW

File: app/Libraries/BitbucketNotification.php [hunk 1]
```diff
@@ -25,6 +25,15 @@ public function __construct($notification)
 
     public function run()
     {
+        $action = $this->notification->header("X-Event-Key");
+        $validActions = ["pullrequest:created"];
+
+        // Only 'pullrequest:created' action is supported at this moment
+        if (!in_array($action, $validActions)) {
+            echo "The action '{$action}' is not valid. Only 'pullrequest:created' action is supported at this moment.\n";
```
Comment (DannyFeliz08) [inline app/Libraries/BitbucketNotification.php]: Hey @DannyFeliz check this

Comment (DannyFeliz) [reply app/Libraries/BitbucketNotification.php]: Check this you too @dawin09

Comment (DannyFeliz) [reply app/Libraries/BitbucketNotification.php]: Check this @gabbanaesteban

Comment (DannyFeliz) [reply app/Libraries/BitbucketNotification.php]: Check this @dawin09 and give me a feedback

Comment (DannyFeliz) [reply app/Libraries/BitbucketNotification.php]: Check this @dawin09 and give me a feedback

**Pull request merged**